physics/ephemeris_test.cpp

```diff
@@ -1111,16 +1111,16 @@ TEST(EphemerisTestNoFixture, DiscreteTrajectoryCompression) {
   trajectory1.WriteToMessage(&message, /*forks=*/{});
   std::string uncompressed;
   message.SerializePartialToString(&uncompressed);
-  EXPECT_EQ(18'433, uncompressed.size());
+  EXPECT_EQ(18'377, uncompressed.size());
 
   std::string compressed;
   auto compressor = google::compression::NewGipfeliCompressor();
   compressor->Compress(uncompressed, &compressed);
 
   // We want a change detector, but the actual compressed size varies depending
   // on the exact numerical values, and therefore on the mathematical library.
-  EXPECT_LE(16'974, compressed.size());
-  EXPECT_GE(16'974, compressed.size());
+  EXPECT_LE(16'935, compressed.size());
+  EXPECT_GE(16'935, compressed.size());
```

Comment:
I suppose things are getting smaller because we lose the metadata header?

Reply:
Yes.  48 bits per series, I think, and then the vagaries of gipfeli.